backends/tfhe-cuda-backend/cuda/src/integer/scalar_shifts.cuh

```diff
@@ -107,4 +109,127 @@ __host__ void host_integer_radix_scalar_shift_kb_inplace(
   }
 }
 
+template <typename Torus>
+__host__ void scratch_cuda_integer_radix_arithmetic_scalar_shift_kb(
+    cuda_stream_t *stream, int_arithmetic_scalar_shift_buffer<Torus> **mem_ptr,
+    uint32_t num_radix_blocks, int_radix_params params, SHIFT_TYPE shift_type,
+    bool allocate_gpu_memory) {
+
+  cudaSetDevice(stream->gpu_index);
+  *mem_ptr = new int_arithmetic_scalar_shift_buffer<Torus>(
+      stream, shift_type, params, num_radix_blocks, allocate_gpu_memory);
+}
+
+template <typename Torus>
+__host__ void host_integer_radix_arithmetic_scalar_shift_kb_inplace(
```

Comment:
Do you think it is worth to add a comment explaining the difference between "arithmetic" and "logic"?